client/settings/payment-methods-selector/index.js

```diff
@@ -0,0 +1,114 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, CheckboxControl, Icon, Modal } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { HorizontalRule } from '@wordpress/primitives';
+
+/**
+ * Internal dependencies
+ */
+import { addSelectedPaymentMethods } from 'data';
+import PaymentMethodIcon from 'settings/payment-method-icon';
+import './style.scss';
+
+const PaymentMethodsSelector = ( props ) => {
+	const { onClose, enabledPaymentMethods = [] } = props;
```

Comment:
The idea here with `enabledPaymentMethods` is to pass a list of string to set which payment methods are selected by default. For example, if `giropay` and `sofort` are already added, this component will receive in this prop: `[ 'giropay', 'sofort' ]`